sdk/typescript/dev/src/index.ts

```diff
@@ -89,7 +89,7 @@ class TypescriptSdkDev {
   /**
    * Bump the TypeScript SDK's Engine dependency.
    *
-   * Example usage: `dagger -m dev call --source=. bump --version v0.10.2 -o .`
+   * Example usage: `dagger call --source=. bump --version v0.10.2 -o .`
```

Comment:
Same as Python, this is not about the root `/dev` module. It's about `sdk/typescript/dev`.